src/Tools/dotnet-user-jwts/test/UserJwtsTests.cs

```diff
@@ -396,4 +395,63 @@ public void Create_GetsAudiencesFromAllIISAndKestrel()
         Assert.Contains("New JWT saved", output);
         Assert.Contains($"Audience(s): http://localhost:23528, https://localhost:44395, https://localhost:5001, http://localhost:5000", output);
     }
+
+    [Fact]
+    public async Task Create_SupportsSettingACustomIssuerAndScheme()
+    {
+        var projectPath = _fixture.CreateProject();
+        var project = Path.Combine(projectPath, "TestProject.csproj");
+        var secretsFilePath = PathHelper.GetSecretsPathFromSecretsId(_fixture.TestSecretsId);
+
+        var app = new Program(_console);
+        app.Run(new[] { "create", "--project", project, "--issuer", "test-issuer", "--scheme", "test-scheme" });
+        var matches = Regex.Matches(_console.GetOutput(), "New JWT saved with ID '(.*?)'");
+        var id = matches.SingleOrDefault().Groups[1].Value;
+        app.Run(new[] { "print", id, "--project", project, "--show-all" });
+        var output = _console.GetOutput();
+
+        Assert.Contains("New JWT saved", output);
+
+        using FileStream openStream = File.OpenRead(secretsFilePath);
+        var secretsJson = await JsonSerializer.DeserializeAsync<JsonObject>(openStream);
+        Assert.True(secretsJson.ContainsKey($"test-scheme:test-issuer:KeyMaterial"));
```

Comment:
I'm probably missing something, but it looks like this should be under `Scheme:test-scheme:test-issuer:KeyMaterial`, see https://github.com/dotnet/aspnetcore/blob/202fafe68774e2a968d8b175c0fd3fb318778903/src/Security/Authentication/Core/src/AuthenticationConfigurationProviderExtensions.cs#L30